Docker/README.md

```diff
@@ -0,0 +1,193 @@
+# Docker support for ROSCon2023Demo
+
+## Prerequisites
+
+* [Hardware requirements of o3de](https://www.o3de.org/docs/welcome-guide/requirements/)
+* Any Linux distribution that supports Docker and the Nvidia container toolkey (see below)
+* At least 60 GB of free disk space
+* Docker installed and configured
+  * **Note** It is recommended to have Docker installed correctly and in a secure manner so that the Docker commands in this guide do not require elevated priviledges (sudo) in order to run them. See [Docker Engine post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) for more details.
+* [NVidia container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+* [rocker](https://github.com/osrf/rocker)
```

Comment:
There are issues running O3DE with rocker, Ubuntu 22, and Nvidia drivers >525 (the Nvidia GPU is not recognized in the docker).

I proposed a fix for that, but it is still in PR: https://github.com/osrf/rocker/pull/277

Maybe it is worth mentioning here.

Reply:
Thats good to know, thanks